apps/masterbots.ai/components/routes/thread/user-thread-panel.tsx

```
@@ -113,12 +113,12 @@ export default function UserThreadPanel({
     setLoading(false)
   }
 
-  React.useEffect(() => {
+  useEffect(() => {
```

Biome

[error] This hook does not specify all of its dependencies: handleThreadsChange

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)